### ISSUE TITLE
Blazor Webassembly Support

### ISSUE THREAD

**Issue by chhinsras** [appwrite/appwrite#1160]

**Comment by TorstenDittmann** [comment]
Hey 👋🏻 

Would a generic .NET SDK for Appwrite be sufficient? Also are both the Client and Server Side be needed?

**Comment by chhinsras** [comment]
Yes. it should be good to support both of server and client side

**Comment by san-coding** [comment]
Hey, I am new to this repo and open source, can I work on this ? I want to make a contribution

**Comment by stnguyen90** [comment]
Related issue: https://github.com/appwrite/sdk-generator/issues/386

**Comment by AathifMahir** [comment]
Dotnet Client SDK would be game changer in my case, Currently I was looking to migrate around couple of big Firebase Project to Appwrite, the only blocker here is Dotnet Client SDK.

**Comment by eldadfux** [comment]
Stale issue, closing down.
